src/main/java/pl/ania/domain/visits/Visit.java

```diff
@@ -0,0 +1,76 @@
+package pl.ania.domain.visits;
+
+import pl.ania.domain.Specialization;
+import pl.ania.domain.doctors.Doctor;
+import pl.ania.security.User;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.util.Date;
+
+@Entity
+public class Visit {
+
+    @Id
+    private String id;
+
+    private Date dateOfVisit;
+
+    private boolean taken;
```

Comment:
to pole jest już bez sensu skoro jest ustawione na true tylko jak jest user != null